Yvand.EntraCP/Yvand.EntraClaimsProvider/EntraIDEntityProvider.cs

```diff
@@ -450,6 +460,44 @@ protected virtual async Task<List<DirectoryObject>> QueryEntraIDTenantAsync(Oper
                         groupsRequestId = await batchRequestContent.AddBatchRequestStepAsync(groupRequest).ConfigureAwait(false);
                     }
 
+                    // List of groups that users must be member of, to be returned to SharePoint
+                    string[] groupsWhichUsersMustBeMemberOfRequestIds = null;
+                    string groupsWhichUsersMustBeMemberOfAny = this.Settings.GroupsWhichUsersMustBeMemberOfAny;
+                    //groupsWhichUsersMustBeMemberOfAny = "c9a94341-89b5-4109-a501-2a14027b5bf0"; // testEntraCPGroup_005 - everyone member
```

Comment:
Should those comments be included in the `master` branch?

Reply:
I may or may not keep them, they can could be useful as hints for debugging in the future